provider/image.go

```diff
@@ -813,13 +813,26 @@ func configureDockerClient(configs map[string]string) (*client.Client, error) {
 
 	// No TLS certificate material provided, create an http client
 	if host != "" {
+		// first, check for ssh host
+		sshopts := []string{}
+		helper, err := connhelper.GetConnectionHelperWithSSHOpts(host, sshopts)
+		if err != nil {
+			return nil, err
+		}
+		if helper != nil {
```

Comment:
I don't understand this bit here, is there ever the case where helper==nil? Oh

```It returns nil without error when no helper is registered for the scheme.```

Reply:
added explanation!